examples/image_generation/diffusion_fast/model_config.yaml

```diff
@@ -0,0 +1,18 @@
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 200
+responseTimeout: 3600
+deviceType: "gpu"
+handler:
+    model_weights: "./Base_Diffusion_model"
```

Comment:
can we change to model_path to align with the other LMI example style?

Reply:
done